plugins/logitech-bulkcontroller/logitech-bulkcontroller.quirk

```diff
@@ -0,0 +1,23 @@
+# SRS TODO
+# Revisit InstallDuration
+# ParentGuid = USB\VID_046D&PID_08A4
+
+[USB\VID_046D&PID_089B]
+Plugin = logitech_bulkcontroller
+Vendor = Logitech
+InstallDuration = 1500
+VersionFormat = triplet
+Flags = rallybar
```

Comment:
I don't think you actually use these flags do you?

Reply:
Removed